CHANGELOG.md

```diff
@@ -5,6 +5,100 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.9.2
+
+_Jul 30, 2025_
+
+We'd like to extend a big thank you to the 21 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌎 Improve French (fr-FR), Hebrew (he-IL) and Polish (pl-PL) locales on the Data Grid
+- 🌎 Improve Korean (ko-KR) locale on the Date and Time Pickers
```

Comment:
```suggestion
- 🌎 Improve Korean (ko-KR) locale on the Date and Time Pickers
- 📈 Add symlog scale to charts
- 📊 Fix bar border radius on Firefox
```


The symlog scale feature isn't that big, not sure if it warrants a minor release.